java/ql/lib/semmle/code/java/security/OgnlInjection.qll

```diff
@@ -3,6 +3,7 @@
 import java
 private import semmle.code.java.dataflow.DataFlow
 private import semmle.code.java.dataflow.ExternalFlow
+private import semmle.code.java.frameworks.MyBatis
```

Comment:
`MyBatis` is already imported in `ExternaFlow`, which means it's already in the global scope. You can remove this.

```suggestion
```

Reply:
`MyBatisInjectionSink` can't be found if I delete that import. It is privately imported in 
https://github.com/github/codeql/blob/main/java/ql/lib/semmle/code/java/dataflow/ExternalFlow.qll#L137, but I guess since it is private, we can't access it in this context.

Reply:
You're absolutely right. I was only thinking about the taint steps, but if you need to directly use a class from `MyBatis` here (in this case, `MyBatisInjectionSink`), you need the import indeed.